<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CreateUserScreen</class>
  <widget class="QWidget" name="CreateUserScreen">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
      </rect>
    </property>
    <property name="styleSheet">
      <string notr="true">QWidget#CreateUserScreen { background-color: #303030; }
QLabel { color: white; font-size: 48pt; }
QLineEdit { background-color: #404040; color: white; border: 2px solid #505050; border-radius: 10px; padding: 15px; font-size: 36pt; }
QPushButton { background-color: #404040; color: white; border: 2px solid #505050; border-radius: 10px; padding: 10px; font-size: 48pt; min-width: 300px; min-height: 120px; }
QPushButton:hover { background-color: #505050; border-color: #606060; }
QPushButton:pressed { background-color: #606060; }</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
        <number>40</number>
      </property>
      <item>
        <widget class="QLabel" name="titleLabel">
          <property name="text"><string>Create User</string></property>
          <property name="alignment"><set>Qt::AlignCenter</set></property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="usernameEdit">
          <property name="placeholderText"><string>Username</string></property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="passwordEdit">
          <property name="placeholderText"><string>Password</string></property>
          <property name="echoMode"><enum>QLineEdit::Password</enum></property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" name="buttonLayout">
          <item>
            <widget class="QPushButton" name="backButton"><property name="text"><string>Back</string></property></widget>
          </item>
          <item>
            <widget class="QPushButton" name="createButton"><property name="text"><string>Create</string></property></widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>